templates/skeleton/app/components/Cart.tsx

```diff
@@ -184,7 +208,7 @@ function CartLineQuantity({line}: {line: CartLine}) {
       <CartLineUpdateButton lines={[{id: lineId, quantity: prevQuantity}]}>
         <button
           aria-label="Decrease quantity"
-          disabled={quantity <= 1}
+          disabled={quantity <= 1 || !!isOptimistic}
```

Comment:
Disable the cart line action buttons when `line.isOptimistic` is true. `isOptimistic` only goes to true when it's an entirely new line. So a new item in the cart that hasn't synced yet to the server. Actions cannot be taken because there is no `lineId` yet. But other optimistic actions, like changing the line quantity with `LinesUpdate`, `line.isOptimistic` stays false. This allows the user to rapidly click the +1 button to add many to the cart. The root `cart.isOptimistic` does get set to `true`. So the dev could still show a generic visual indicator for the whole cart until all actions are resolved.